components/engine/engine-camel/src/main/java/org/eclipse/dirigible/components/engine/camel/invoke/IntegrationMessage.java

```diff
@@ -0,0 +1,41 @@
+package org.eclipse.dirigible.components.engine.camel.invoke;
+
+import org.apache.camel.Message;
+
+import java.util.Map;
+
+public class IntegrationMessage {
```

Comment:
Where's the TypeScript interface for this class?

Reply:
I removed it by mistake it's added back in the PR